src/runtime.js

```diff
@@ -219,6 +235,37 @@ export function adopt(global) {
     // go out of scope here, but just making sure.
     preregisteredElements.length = 0;
   });
+
+  // For iOS we need to set `cursor:pointer` to ensure that click events are
+  // delivered.
+  if (platformFor(global).isIos()) {
+    setStyle(global.document.documentElement, 'cursor', 'pointer');
```

Comment:
Correct. Just moved it - seemed in the wrong place and was easier to refactor `installStyle`